.github/workflows/srtool-build.yml

```diff
@@ -53,35 +52,28 @@ jobs:
       - name: Show Runtime information
         shell: bash
         run: |
-          subwasm info ${{ steps.srtool_build.outputs.wasm }}
-          subwasm info ${{ steps.srtool_build.outputs.wasm_compressed }}
-          subwasm --json info ${{ steps.srtool_build.outputs.wasm }} > ${{ matrix.runtime }}-info.json
-          subwasm --json info ${{ steps.srtool_build.outputs.wasm_compressed }} > ${{ matrix.runtime }}-compressed-info.json
+          subwasm info --json  ${{ steps.srtool_build.outputs.wasm_compressed }} > ${{ matrix.runtime }}-compressed-info.json
 
       - name: Extract the metadata
         shell: bash
         run: |
-          subwasm meta ${{ steps.srtool_build.outputs.wasm }}
-          subwasm --json meta ${{ steps.srtool_build.outputs.wasm }} > ${{ matrix.runtime }}-metadata.json
+          subwasm meta --format=json+scale ${{ steps.srtool_build.outputs.wasm_compressed }}
 
       - name: Check the metadata diff
         shell: bash
         run: |
           subwasm get -o ${{ matrix.runtime }}-live.wasm wss://${{ matrix.runtime }}.kilt.io
-          subwasm diff --no-color ${{ matrix.runtime }}-live.wasm  ${{ steps.srtool_build.outputs.wasm }} || \
+          subwasm diff --no-color ${{ matrix.runtime }}-live.wasm  ${{ steps.srtool_build.outputs.wasm_compressed }} || \
             echo "Subwasm call failed, check the logs. This is likely because ${{ matrix.runtime }} is not known by subwasm" | \
             tee ${{ matrix.runtime }}-diff.txt
 
       - name: Archive Subwasm results
-        uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3
+        uses: actions/upload-artifact@v4
         with:
           name: ${{ matrix.runtime }}-runtime
           path: |
-            ${{ matrix.runtime }}-info.json
             ${{ matrix.runtime }}-compressed-info.json
-            ${{ matrix.runtime }}-metadata.json
```

Comment:
But not it seems we don't store the metadata at all anymore?

Reply:
Added back